docs/xml/xml_data_best_practices.rst

```diff
@@ -0,0 +1,142 @@
+| CandidateSelection                     | cs                                    |
++----------------------------------------+---------------------------------------+
+| ContactInformation                     | ci                                    |
++----------------------------------------+---------------------------------------+
+| ElectionAdministration                 | ea                                    |
++----------------------------------------+---------------------------------------+
+| ElectoralDistrict                      | ed                                    |
++----------------------------------------+---------------------------------------+
+| HoursOpen                              | hours                                 |
++----------------------------------------+---------------------------------------+
+| Locality                               | loc                                   |
++----------------------------------------+---------------------------------------+
+| Office                                 | off                                   |
++----------------------------------------+---------------------------------------+
+| OrderedContest                         | oc                                    |
++----------------------------------------+---------------------------------------+
+| Party                                  | par                                   |
++----------------------------------------+---------------------------------------+
+| PartyContest                           | pc                                    |
++----------------------------------------+---------------------------------------+
+| PartySelection                         | ps                                    |
++----------------------------------------+---------------------------------------+
+| Person                                 | per                                   |
++----------------------------------------+---------------------------------------+
+| PollingLocation                        | pl                                    |
++----------------------------------------+---------------------------------------+
+| RetentionContest                       | rc                                    |
++----------------------------------------+---------------------------------------+
+| State                                  | st                                    |
++----------------------------------------+---------------------------------------+
+| StreetSegment                          | ss                                    |
++----------------------------------------+---------------------------------------+
+| Source                                 | src                                   |
++----------------------------------------+---------------------------------------+
+| Election                               | ele                                   |
```

Comment:
Is there any particular reason that this list is in almost-but-not-quite alphabetical order?